website_mrp_bom/tests/test_website_mrp_bom.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from mock import patch
+from odoo.tests.common import SavepointCase
+from ..controllers.main import WebsiteMrpBomPage
+
+
+class TestController(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestController, cls).setUpClass()
+        cls.uom_unit = cls.env.ref('product.product_uom_unit')
+
+        cls.product_sale_ok = cls.env['product.template'].create({
```

Comment:
you can simply create the product straight and get them in one shot w/out having to search